README.md

```diff
@@ -69,6 +69,9 @@ Requirements
   - `download_path`: Local folder for downloading artifacts
     default: `/tmp/`
 
+  - `windows_temp_path`: Windows local folder for downloading artifacts
+    default: `C:\Windows\Temp\`
```

Comment:
should be changed to lookup tmp from env variables

Reply:
fixed